tests/MartinGeorgiev/Doctrine/Fixtures/Entity/ContainsDates.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\MartinGeorgiev\Doctrine\Fixtures\Entity;
+
+/**
+ * @Entity
+ */
+class ContainsDates extends Entity
+{
+    /**
+     * @var \DateTime
+     *
+     * @Column(type="date")
+     */
+    public $date1;
+
+    /**
+     * @var \DateTimeImmutable
+     *
+     * @Column(type="date_immutable")
+     */
+    public $date2;
```

Comment:
Do you think the data types can be enforced nativuly to the properties instead of using doc blocks?

Reply:
Indeed. However, I just remembered that the package supports PHP 7.2 and 7.3, I completely zoomed out that the support for 7.4 is only in a local branch ha-ha... Ignore my silly comment.

Reply:
:))